Program.markdown

```diff
@@ -152,10 +173,15 @@ Patenter, som statlig garanterte monopoler, skaper en kunstig innskrenkning av d
 
 Forskning som er offentlig finansiert (inkludert rådata, programkode og metoder) skal ligge åpent på Internett. Det tjener ikke samfunnet som helhet at forskning foregår bak lukkede dører uten mulighet for innsyn for andre i rådata, metoder, og programkode som er brukt som grunnlag for konklusjoner i forskningen.
 
+I den vestlige verden har opphavsrett blitt en alvorlig trussel mot fri kommunikasjon, uavhengig om folk gjør noe lovlig eller ulovlig. Opphavsrett blitt brukt for å rettferdiggjøre sensur, spionasje og innføring av restriksjoner på Internett. Piratpartiet er ikke imot opphavsrett, men det er åpenbart behov for å oppdaterte lovverket.
+
+Piratpartiet mener at utdanningssystemet må oppdateres. Samfunn og utdanningssystemet må være i harmoni fra barnehage til høyere utdanning. Formålet med utdanningssystemet er å lære folk hvordan samfunnet fungerer og hvordan ny kunnskap blir skapt. Altfor mange menneske med høyere utdanning står uten muligheter til å finne arbeid i sitt fagfelt. Dette vil Piratpartiet endre ved å gjøre utdanninger på alle nivå mer varierte, fleksible, koblet til Internett og direkte koblet til samfunnet. Piratpartiet vil også at seksualitet skal være eget fag i grunnskolen, der det blir lagt vekt på gjensidig respekt, kommunikasjon og informert samtykke.
+
+Utdanning skal gi pensjonspoeng. Folk som tar høyere utdanning eller videreutdanning kommer senere ut i arbeidslivet enn de som ikke tar det.
```

Comment:
Kopiert riktig. 
Forslag for LM2017: Bind saman setningane på linje 180 med ", siden"